osmaxx/excerptexport/templates/excerptexport/partials/export_detail_string.html

```diff
@@ -1,7 +1,7 @@
 {% load i18n capture_as %}{{ export.get_file_format_display }} /
-{% firstof export.finished_at|date:"Y-m-d H:i:s" export.updated_at|date:"Y-m-d H:i:s" as finished_date %}
```

Comment:
Ah, [this](https://mounirmesselmeni.github.io/2014/11/06/date-format-in-django-admin/) works. :-)